### ISSUE TITLE
Specify deno-lambda version

### ISSUE THREAD

**Issue by hayd** [lucacasonato/now-deno#4]
Atm now-deno uses the latest release but I think it would be good for users to (optionally?) pass a version number for the deno-lambda release to use.

(In the future this will match the deno version.)

**Comment by hayd** [comment]
Should this be config.denoVersion? Is that the way to do this?

**Comment by lucacasonato** [comment]
I dont think that is the way. I think that is the legacy way for the ['builds' property](https://zeit.co/docs/configuration#project/builds) - the new 'functions' property does not support config I think. The Node.js runtime [uses engines.node in package.json](https://zeit.co/docs/runtimes#official-runtimes/node-js/node-js-version) and [Go is just pinned to latest 1.x](https://zeit.co/docs/runtimes#official-runtimes/go/go-version). We can probably pin to latest and let user specify a specific version via the env var DENO_VERSION - that should then 1:1 map to deno-lambda version. That env var can then be specified using [`build.env` in now.json](https://zeit.co/docs/configuration#project/build-env).

**Comment by lucacasonato** [comment]
I'll wait for https://github.com/hayd/deno-lambda/issues/20 to be done before implementing this.

**Comment by stephengfriend** [comment]
Mind if I grab this? I was working on delivering a PR for `now dev` but noticed it landed a little bit ago.

**Comment by lucacasonato** [comment]
Sure. Just be aware I won't merge it until https://github.com/hayd/deno-lambda/issues/20 is closed.

**Comment by stephengfriend** [comment]
That's totally fair. I wrote the tests to be compatible with it and I'm giving it a run-through, as well.

**Comment by hayd** [comment]
https://github.com/hayd/deno-lambda/issues/20 is closed now: releases will match the deno version.

**Comment by lucacasonato** [comment]
@stephengfriend Do you still want to take this? If you don't want to anymore, I'll take it.

**Comment by stephengfriend** [comment]
Hey, thanks for checking in. I almost forgot about it. Yes, I'll take a look today.
